src/styles/main.scss

```diff
@@ -1,3 +1,74 @@
+$row-width: 100px;
```

Comment:
@Fluebubble  I read your comment about putting everything in different files, but it is a very good practice to put variables and functions separately from all rules, this will increase the readability and maintainability of the code. Therefore, it would be cool to take it out in a separate file 😎